applications/Web3Go.md

```diff
@@ -191,7 +191,7 @@ Web3Go team members have strong engineering background: Han Zhao, Yifei Wu and M
 
 ### Milestone 2  — More databoards, fully-automatic chart generation
 
-* **Estimated Duration:** 1 month
+* **Estimated Duration:** 8 month
 * **FTE:**  2
 * **Costs:** 15,000 USD
```

Comment:
```suggestion
* **Costs:** 9,000 USD
```

This is what I'd suggest, based on the fact that the two removed deliverables would have been the most impactful ones of this milestone. Feel free to commit this or suggest otherwise.

Reply:
it is ok for us. thanks for that!